tools/accuracy_checker/accuracy_checker/annotation_converters/coco_facial_landmarks.py

```diff
@@ -0,0 +1,87 @@
+"""
+Copyright (c) 2018-2021 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import numpy as np
+from .format_converter import FileBasedAnnotationConverter, ConverterReturn
+from ..representation import Face98LandmarksAnnotation
+from ..utils import read_xml, check_file_existence
+from ..config import PathField
+from xtcocotools.coco import COCO
```

Comment:
I'm not sure that we need this heavy dependency (with requirements for C++ extension compilation) just for json file parsing. Can we avoid it usage?

Reply:
It's origin format annotation for the training in mmpose. And I don't want to convert it to another. Is it too difficult to add? It's very useful module to parse coco-like annotations.

Reply:
it is problematic for windows in environment where no VS :(
and beside that internally we use pycocotools for metrics, they may conflict...

Reply:
I do not ask to convert, I just ask to parse json using json. is it too difficult for you?

Reply:
BTW, here no wheel for windows and macos at all https://pypi.org/project/xtcocotools/#files

Reply:
you also can reuse conversion code from `mscoco_keypoints` converter, it works with the same data format

Reply:
"I do not ask to convert, I just ask to parse json using json. is it too difficult for you?" 

Of course no. Ok. 

I received answer, why we can use it:
"it is problematic for windows in environment where no VS :(
and beside that internally we use pycocotools for metrics, they may conflict..."

Reply:
Done) Don't worry! Be happy))))

Reply:
Thank you!